test/unit/app/models/test_models.py

```diff
@@ -249,6 +249,81 @@ def test_new_credit_transfer_rejected(create_credit_transfer):
     assert create_credit_transfer.resolution_message is not None
 
 
+def test_new_credit_transfer_check_sender_txns_limits(create_credit_transfer):
+    """
+    GIVEN a CreditTransfer model
+    WHEN a new credit transfer is created
+    THEN check the correct check_sender_txns_limits apply
+    """
+    from server.utils.transaction_limits import limits
+    from server.models.kyc_application import KycApplication
+    from server.models import token
+
+    # Sempo Level 0 limits (payment only)
+    assert create_credit_transfer.check_sender_txns_limits() == [limit['rule'] for limit in limits[:2]]
+
+    # Check Sempo Level 1 limits (payment only)
+    create_credit_transfer.sender_user.is_phone_verified = True
+    assert create_credit_transfer.check_sender_txns_limits() == [limit['rule'] for limit in limits[3:5]]
```

Comment:
the issue with taking a sub-list is that if we added say a new rule at the front of the list, it breaks all of these in a kinda irritating way.

It'd require a bit of shuffling to fix though (perhaps by converting to a dict with unique keys for each limit), so perhaps we just ignore for now lol

Reply:
The other option is to filter by the rule name and relevant transfer type?

i.e. 
`[limit['rule'] for limit in LIMITS[-1:] if rule_name == limit['rule']['name'] ]`